src/shared-components/button/components/button/index.tsx

```diff
@@ -2,26 +2,17 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { useTheme } from 'emotion-theming';
 
-import Loader from './shared-components/loader';
```

Comment:
Only change in this file is the relative import paths changing, and removing the importing/exporting of other non-Button buttons